BackofficeBundle/Resources/translations/role.en.yml

```diff
@@ -1,6 +1,9 @@
 open_orchestra_role:
     role_access_general_node: Manager the transversal pages
     role_access_tree_node: Manage pages
+    role_access_create_node: Create pages
+    role_access_update_node: Update pages
+    role_access_delete_node: Delete pages
```

Comment:
In the roles list given by @nbouquet , all roles don't use the plural forms. Here it should be ``Create a page``, and so on.